apps_wishlist.md

```diff
@@ -171,6 +171,7 @@ The following list is a compiled wishlist of applications that would be nice-to-
 - [Simone](https://github.com/Kloadut/Simone)
 - [SocialHome](https://github.com/jaywink/socialhome) (Socialhome is best described as a federated personal profile with social networking functionality) (https://socialhome.network)
 - [Sonarr](https://sonarr.tv)
+- [smokeping](https://packages.debian.org/buster/smokeping)   
```

Comment:
```suggestion
- [smokeping](https://packages.debian.org/buster/smokeping)   / [github](https://github.com/oetiker/SmokePing)
```

Reply:
Can I suggest http://www.smokeping.org instead of the github link ? Or is there a reason to point to the github project ?

Reply:
I'm not packager, but just two objectives : 
+ For documentation contributors : see if the app is still maintained.
+ For potentital packager, general informations

I add link to wishlist for apps haven't "technical" informations (ref : #1095, particulary [this commit](https://github.com/YunoHost/doc/pull/1095/commits/2d59689d64783e16b4eb1e8a0d19debd200c6670)). 

Choose the most relevant resource. =)